src/components/userProfile/userProfile.css

```diff
@@ -0,0 +1,43 @@
+.user-profile {
```

Comment:
as we discussed on Wednesday, we should choose some kind of organizing scheme for CSS and be consistent. Expect this will be a design decision, but the more CSS we write without deciding the bigger a PITA it'll be to change.